conjure-undertow-processor/src/test/java/com/palantir/conjure/java/undertow/processor/sample/ContentTypeFilter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.processor.sample;
+
+import com.palantir.conjure.java.undertow.annotations.Handle;
+import com.palantir.conjure.java.undertow.annotations.HttpMethod;
+import javax.ws.rs.core.MediaType;
+
+public interface ContentTypeFilter {
+
+    @Handle(method = HttpMethod.POST, path = "/contentTyp")
+    @Handle.Accepts(MediaType.TEXT_PLAIN)
+    void contentType(@Handle.Body String body);
```

Comment:
This is a bit awkward because the `Handle.Body(deserializer=<VALUE>)` needs to agree with the `Accepts` values, but we don't have anything in place which forces that to be true. So in this case we'd fail unless we receive `text/plain` content, but the body default deserializer would fail to accept most input because it requires json, smile, or cbor.